components/alibi-explain-server/Dockerfile

```diff
@@ -60,6 +60,7 @@ RUN yum -y update-minimal --security --sec-severity=Important --sec-severity=Cri
 RUN yum remove -y nodejs httpd
 
 COPY --from=builder /microservice /microservice
+COPY --from=builder /licenses /licenses
```

Comment:
Were the licenses not generated before? Normally we do have the requirement to include all licenses inside the image, but I'm assuming AD never actually had the scipts to extract all the licenses

Reply:
It was missed when introduced double-stage build. That happened in master after 1.11.2 and it was just a mistake when introducing poetry in alibi explain server that I spotted now when doing detect server. 

So we always had these in what we released, master was missing it for some time, and this fixes it.